<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSystemInit</class>
 <widget class="QDialog" name="DlgSystemInit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_title" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>33</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lab_Ico">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_Title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>系统初始化</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_menu" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnMenu_Close">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>关闭</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>采集卡选项</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>采集卡类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxCameraType">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>采集卡数目：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxCameraNum">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>通讯方式</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>通讯类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxIOType">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>305</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonOK">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>确  认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCancel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>取  消</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
